web/src/components/base/ToggleButton.tsx

```diff
@@ -35,7 +36,7 @@ type ToggleButtonProps = {
 const ToggleButton = ({ text, onClick, className, value, ...props }: ToggleButtonProps): ReactElement => (
   <StyledButton
     color='primary'
-    sx={{ textTransform: 'inherit' }}
+    sx={{ textTransform: 'inherit', borderRadius: '18px !important' }}
```

Comment:
🤔 Why is the `textTransform: 'inherit'` necessary? I can't really spot a difference 🙈